app/views/play/CampaignView.coffee

```diff
@@ -772,19 +769,20 @@ module.exports = class CampaignView extends RootView
       window.tracker?.trackEvent 'Clicked Start Level', category: 'World Map', levelID: levelSlug
 
   onClickCourseVersion: (e) ->
-    levelSlug = $(e.target).parents('.level-info-container').data 'level-slug'
     courseID = $(e.target).parents('.course-version').data 'course-id'
-    courseInstanceID = $(e.target).parents('.course-version').data 'course-instance-id'
-    url = "/play/level/#{levelSlug}?course=#{courseID}&course-instance=#{courseInstanceID}"
-    Backbone.Mediator.publish 'router:navigate', route: url
+    levelElement = $(e.target).parents('.level-info-container')
+    @startLevel levelElement, courseID, @getQueryVariable('course-instance')
+
+  onClickInventoryModalButton: (e) ->
+    @startLevel $("<div>").data("level-slug","dungeons-of-kithgard").data("level-path","level").data("level-name","Dungeons of Kithgard"), @courseInstance.get('courseID'), @getQueryVariable('course-instance'), true
```

Comment:
Trying to figure out a way to do this better, but it would help to know what behavior you're shooting for.